substratevm/src/com.oracle.svm.core.posix/src/com/oracle/svm/core/posix/PosixJavaNetSubstitutions.java

```diff
@@ -94,17 +94,246 @@ private PosixJavaNetSubstitutions() {
 @TargetClass(className = "java.net.PlainDatagramSocketImpl")
 @Platforms({Platform.LINUX.class, Platform.DARWIN.class})
 final class Target_java_net_PlainDatagramSocketImpl {
+    @Alias boolean connected;
 
     @Substitute
```

Comment:
It would be good to tell the reader where this code comes from.  Something like
```
/* Translated from: {jdk8u}/src/solaris/native/java/net/PlainDatagramSocketImpl.c */
// JNIEXPORT void JNICALL
// Java_java_net_PlainDatagramSocketImpl_bind0(JNIEnv *env, jobject this,
//                                            jint localport, jobject iaObj) {
```
If that is in fact where it comes from.  That at least gets me to the correct file and method.  Then I can start following lines even if they do not have line numbers.  Extra credit for line numbers.